.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: Create new release
+
+on:
+  schedule:
+    # Runs "At 23:59 every night" (UTC)
+    - cron: '59 23 * * *'
+
+jobs:
+  fetch-changelog:
```

Comment:
Can remove this job. `CHANGELOG.md` is available after checkout in `create-new-release` job